test/parallel/test-tls-basic-validations.js

```diff
@@ -56,3 +56,17 @@ assert.throws(() => tls.createSecurePair({}),
   assert(buffer.equals(Buffer.from('abcd')));
   assert(out.NPNProtocols.equals(Buffer.from('efgh')));
 }
+
+{
+  const buffer = new Uint8Array([...Buffer.from('abcd')]);
```

Comment:
For this case, indeed it would work. Passing a Buffer to Uint8Array makes the Uint8Array use the identical underlying ArrayBuffer, which at times may not be desirable. But in this case we are creating a new Buffer every time so it should be fine.